package/MDAnalysis/analysis/lineardensity.py

```diff
@@ -81,15 +81,18 @@ class LinearDensity(AnalysisBase):
        print(ldens.results.x.pos)
 
 
-    Alternatively, the other types of groupings can be selected using the ``grouping``
-    keyword. For example to calculated the density based on the :class:`ResidueGroup`s
-    of the system:
-    
+    Alternatively, the other types of groupings can be selected using the
+    ``grouping`` keyword. For example to calculated the density based on
```

Comment:
Minor nitpick that this should be calculate rather than calculated - but otherwise lgtm!